core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.js

```diff
@@ -0,0 +1,691 @@
+    const urlRegex = /http:\/\/localhost:8181\/create\/\w*(\/\w*\/?\w*)?/;
+    if (this.page.url().match(urlRegex) === null) {
+      throw new Error (
+        'The Create Lessons button does not display ' +
+        'the Exploration Editor page!');
+    } else {
+      showMessage(
+        'The Create Lessons button displays the Exploration Editor page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickExploreLessonsButtonInAbout() {
+    await this.clickOn(exploreLessonsButton);
+    if (this.page.url() !== mathClassroomUrl) {
+      throw new Error (
+        'The Explore Lessons button does not open the Math Classroom page!');
+    } else {
+      showMessage(
+        'The Explore Lessons button opens the Math Classroom page.');
+    }
+  }
+
+  /**
+   * Function to click the The Oppia Foundation button in the About Menu
+   * on navbar and check if it opens The About Foundation page.
+   */
+  async clickAboutFoundationInAboutMenuOnNavbar() {
+    await this.clickOn(navbarAboutTab);
+    await this.clickOn(navbarAboutTabAboutFoundationButton);
+    await this.page.waitForSelector(aboutFoundationClass);
+    const displayedh1 = await this.page.$eval(
+      aboutFoundationClass,
+      element => element.innerText
+    );
+    if ((this.page.url() !== aboutFoundationUrl) &&
+      (displayedh1 !== 'THE OPPIA FOUNDATION')) {
+      throw new Error (
+        'The Oppia Foundation button in About Menu on navbar ' +
+        'does not open the About Foundation page!');
+    } else {
+      showMessage(
+        'The Oppia Foundation button in About Menu on navbar ' +
+        'opens the About Foundation page.');
+    }
+  }
+
+  /**
+   * Function for click the 61 million children button
+   * in the About Foundation page and check if it opens the right page.
+   */
+  async click61MillionChildrenButtonInAboutFoundation() {
+    await this.page.waitForSelector(millionsOfContentId);
+    const buttonText = await this.page.$eval(
+      millionsOfContentId,
+      element => element.getElementsByTagName('a')[0].textContent
+    );
+
+    if (buttonText !== '61 million children') {
+      throw new Error (
+        'The 61 Million Children button does not exist!');
+    } else {
+      await this.page.$eval(
+        millionsOfContentId,
+        element => element.getElementsByTagName('a')[0].click()
+      );
+      if (this.page.url() !== _61MillionChildrenUrl) {
+        throw new Error (
+          'The 61 Million Children button does not open the right page!');
+      } else {
+        showMessage(
+          'The 61 Million Children button opens the right page.');
+      }
+    }
+  }
+
+  /**
+   * Function for click the even those who are in school button
+   * in the About Foundation page and check if it opens the right page.
+   */
+  async clickEvenThoseWhoAreInSchoolButtonInAboutFoundation() {
```

Comment:
Again, "link" seems more appropriate over "button" for the same reasons mentioned previously.